test/jsonapi-response-builder.test.js

```diff
@@ -1,11 +1,13 @@
 const test = require('tape');
 const config = require('../config');
 const buildJSONResponse = require('../lib/jsonapi-response.js');
-const exampleGETResponse = require('./fixtures/example-get-response.json');
+const exampleObjResponse = require('./fixtures/example-get-response-object.json');
+const exampleDocResponse = require('./fixtures/example-get-response-document.json');
 
-test('Building the JSON Response', function (t) {
-  t.plan(11);
-  const JSONAPIResponse = buildJSONResponse(exampleGETResponse, config);
+test('Building the JSON Response for an object', function (t) {
```

Comment:
I'd recommend naming your tests in terms of what they "should" or "should not" do. It encourages you to narrow down what you're testing, be a bit more descriptive about what the test is testing and split apart big tests like this into smaller chunks that test a specific aspects.

For instance, I'd be tempted to split this into multiple tests to test that the builder adds the fields required for a jsonapi response -

* "Should create data field" - make sure data is present and has required attributes
* "Should create relationships within data" - make sure terms, maker and places relations are added
* "Should create included field" - make sure included relationships are in the response

I'd then suggest another file for documents, and people that does the same.

Perhaps think of the filename as the "suite", `test()` as the test and `t` as assertions. Right now you seem to have the `test()` as the suite and your assertions (`t`) as the tests.